doc.go

```diff
@@ -99,7 +99,7 @@ User's security responsibilities
 
 The consumer is responsible for
```

Comment:
Is “consumer” the right word here? It may be common usage (in go packages?), but it doesn’t seem quite right to me.

Reply:
I never know what the right term is used for this stuff. I once heard @rickfillion use the word "consumer" in a way that felt like this, so I thought that that is the go to phrase for the user of an API.  I'm happy to change it if told otherwise.